sympy/stats/tests/test_continuous_rv.py

```diff
@@ -619,20 +701,36 @@ def test_triangular():
     a = Symbol("a")
     b = Symbol("b")
     c = Symbol("c")
+    x = Symbol("x")
 
     X = Triangular('x', a, b, c)
     assert density(X)(x) == Piecewise(
                  ((2*x - 2*a)/((-a + b)*(-a + c)), And(a <= x, x < c)),
                  (2/(-a + b), x == c),
                  ((-2*x + 2*b)/((-a + b)*(b - c)), And(x <= b, c < x)),
                  (0, True))
+    #Tests characteristic_function
+    assert characteristic_function(X)(x) == \
+           (2*(-a + b)*exp(I*c*x) - 2*(-a + c)*exp(I*b*x) - 2*(b - c)*exp(I*a*x))/(x**2*(-a + b)*(-a + c)*(b - c))
 
+    #Tests moment_generating_function
+    assert moment_generating_function(X)(x) == \
+    (-2*(-a + b)*exp(c*x) + 2*(a + c)*exp(b*x) + 2*(b - c)*exp(a*x))/(x**2*(-a + b)*(-a + c)*(b - c))
 
 def test_quadratic_u():
+    from sympy import simplify
```

Comment:
As you have commented out the usage of simplify, please remove it (or import it closer to the outcommented part, although it is better to not use simplify in tests if it can be avoided).

Reply:
The test was not exactly working, so I commented it out to see that other tests worked fine in the build and I could find a better alternative for the test.

I'll fix this asap.